src/website/src/app/documentation/demos/header/header-types.demo.html

```diff
@@ -144,7 +144,7 @@
             </div>
             <div class="header-actions">
               <a href="javascript://" class="nav-link nav-icon-text">
-                <clr-icon shape="user"></clr-icon>
+                <clr-icon shape="user" aria-label="user"></clr-icon>
                 <span class="nav-text">username</span>
               </a>
```

Comment:
If the element has text already defined. This example it can be redundant and the screen reader would read "user username". In this case the icon is decorative https://clarity.design/icons/accessibility